service/host.go

```diff
@@ -164,6 +164,7 @@ func (s *manager) shutdown(err error, reason string, exitCode *int) (bool, error
 	}
 
 	// Flush tracing buffers
+	// GFM-378 uhttp.Client is not safe to send requests anymore
```

Comment:
From Alex - "Clients for YARPC are created with a dispatcher and we stop it in modules
before, so it should be ok(error I assume)."

Yes but it's not a module. Clients are not necessarily aware that dispatcher comes from initializing the YARPC module. It is fetched separately from module initialization.